### ISSUE TITLE
TypeError: vue_1.default is not a constructor

### ISSUE THREAD

**Issue by Tlopasha** [DevExpress/devextreme-vue#293]
# TypeError: vue_1.default is not a constructor

**Package versions:**

    devexteme version: 19.1.4
    devextreme-vue version: 19.1.4

**Steps to reproduce:**
 
[Codesandbox](https://codesandbox.io/s/item-selection-and-customization-devextreme-tree-view-1655f)
  

**Current behavior:**

 **error - TypeError: Cannot read property 'add' of undefined** appearce in the Console 
and it cause when the additional custom item template are added.
```
       <div
        slot="item"
        slot-scope="{ data: value }"
      >{{ value.text + (value.price ? ' ($' + value.price + ')' : '') }}</div>
```

**Expected behavior:**
No errors

**Comment by dxvladislavvolkov** [comment]
Fixed #286

**Comment by jncalderon** [comment]
the error continues, for use templates datagrid and list. Im using 19.1.4 also
